packages/cli-plugin-metro/src/commands/bundle/bundleCommandLineArgs.ts

```diff
@@ -102,6 +102,7 @@ export default [
     name: '--unstable-transform-profile <string>',
     description:
       'Experimental, transform JS for a specific JS engine. Currently supported: hermes, hermes-canary, default',
+    default: 'default',
```

Comment:
Note: `unstableTransformProfile` was aligned to be a non-null value, with this default value set here. This matches the internally defined default in Metro.

https://github.com/facebook/metro/blob/4e5261cca6fd84b7ce75f58b038b397b84a4b038/packages/metro/src/Server.js#L1402